### DISCUSSION TITLE
rio.write_crs returns 0 for spatial_ref

### DISCUSSION THREAD

**Discussion by shuai-zhou** [corteva/rioxarray#444] [Q&A] [answered]
Following @snowman2 recommendation [here](https://gis.stackexchange.com/questions/354782/masking-netcdf-time-series-data-from-shapefile-using-python), the `rio.write_crs` gave 0 for `spatial_ref` (see the attached screenshot) on my data. I was wondering what does 0 means in this context, is this the expected result?

![Snipaste_2021-12-26_15-47-40](https://user-images.githubusercontent.com/22311821/147419734-ef1540a5-5692-4a3e-b491-53b777414065.png)

**Comment by snowman2** [comment, marked as answer]
This should help clarify: https://corteva.github.io/rioxarray/stable/getting_started/crs_management.html

Note: CRS information is stored in spatial_ref.attrs

**Comment by shuai-zhou** [comment]
Thanks for the clarification.
